src/components/DropDown/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './styles.scss';
+
+const DropDown = ({ text, onClick }) => {
+    return (
+        <button onClick={onClick}>
+            {text}
+        </button>
+    );
+};
+
+DropDown.propTypes = {
+    text: PropTypes.string,
+    onClick: PropTypes.func
+}
```

Comment:
I don't think this matches the components diagram, when rendered it shows a button with text, instead it should open like a menu that you can select the options.

![image](https://user-images.githubusercontent.com/35865469/104036581-552d6900-51b2-11eb-9fd8-bd8061947d1d.png)

I would implement this using [React bootstrap dropdowns](https://react-bootstrap.github.io/components/dropdowns/) so that we don't get too convoluted with details

Reply:
For layout purpose, we decided to treat the Dropdown like a simple button element. In the future, we'll change it to what it actually is.